tests/listobject_test.py

```diff
@@ -1,32 +1,205 @@
-from tests.utility import *
 from unittest import *
 import botocore
 import botocore.session
-
+from libs.execute import *
+from libs.command import *
+from libs.utility import *
+from unittest import *
+from datetime import datetime
 
 class ListObject_Test(TestCase):
+
+    # ======== Construction, setUp, tearDown =========
+
+    def __init__(self, *args, **kwargs):
+        self.bucket_irods_path = '/tempZone/home/alice'
+        self.bucket_name = 'alice-bucket'
+        self.key = 's3_key2'
+        self.secret_key = 's3_secret_key2'
+        self.s3_bridge_url = 'http://localhost:8080'
```

Comment:
Is the plan to make these configurable in the future?

Reply:
Maybe.  I am not sure once I have everything wired together.

Reply:
All good.

Please capture in an issue and leave a comment referencing it here. We can circle back to this in a later PR.